examples/binding-patterns-in-match-statement/binding_patterns_in_match_statement.metatags

```diff
@@ -0,0 +1,2 @@
+description: This BBE demonstrates the `spread` operator in binding patterns in a `match` statement.
```

Comment:
```suggestion
description: This BBE demonstrates the rest binding pattern in binding patterns in a `match` statement.
```

Reply:
Same comment re: this being an introduction to binding patterns in a match statement rather than just the rest binding pattern.